frontend/components/Domain/ShoppingList/ShoppingListItemEditor.vue

```diff
@@ -128,6 +128,17 @@ export default defineComponent({
         context.emit("input", val);
       },
     });
+
+    watch(
+      () => props.value.food,
+      (newFood) => {
+        console.log(newFood?.label);
```

Comment:
@michael-genson was this a debugging log? Does it benefit us in production?

Reply:
Oops good catch, yup that's a debug log. I'll remove that